packages/api-rx/src/observable/subject.ts

```diff
@@ -4,9 +4,7 @@
 
 import { ApiInterface$Section } from '@polkadot/api/types';
 
-import { BehaviorSubject } from 'rxjs/BehaviorSubject';
-import { Observable } from 'rxjs/Observable';
-import { Subscriber } from 'rxjs/Subscriber';
+import { BehaviorSubject , Observable, Subscriber } from 'rxjs';
```

Comment:
the linter doesn't yell on this space?

Reply:
For some reason not. However, I'm yelling :) Fixing...